metrics/scripts/grafana-provisioning/dashboards/testnet-monitor.json

```diff
@@ -9481,6 +9481,82 @@
             ]
           ],
           "tags": []
+        },
+        {
```

Comment:
And, this is my first `grafana` thing! I've run it for the first time locally and tested this new config!